bin/shards

```diff
@@ -5,9 +5,11 @@
 
 set -euo pipefail # exit on any error, don't allow undefined variables, pipes don't swallow errors
 
-# Export a SHARDS_INSTALL_PATH for the current directory.
-SHARDS_INSTALL_PATH="$HOME/.shards/$(basename "$PWD")"
-export SHARDS_INSTALL_PATH
+if [ -z "${INSTALL_SHARDS_LOCALLY+x}" ] ; then
```

Comment:
This allows running `INSTALL_SHARDS_LOCALLY=1 shards install` to install to `./lib/` (the usual default). I was having some trouble with the `lucky` project getting specs to run when installing to `~/.shards` (I think because of Docker-related issues, since the `~/.shards/lucky` shards weren't being copied into the Docker image), which is why I added this option.